olm-catalog/serverless-operator/generate_csv.sh

```diff
@@ -0,0 +1,72 @@
+# !/bin/bash
```

Comment:
Shellcheck report:

```
Line 1:
# !/bin/bash
 ^-- SC1115: Remove spaces between # and ! in the shebang.
 
Line 65:
echo $template $target
     ^-- SC2086: Double quote to prevent globbing and word splitting.
               ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean:
echo "$template" "$target"
 
Line 68:
cp $template $target
   ^-- SC2086: Double quote to prevent globbing and word splitting.
             ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean:
cp "$template" "$target"
```